php/class-admin.php

```diff
@@ -199,9 +220,11 @@ public function enqueue_assets() {
 				'material-admin-js',
 				'materialDesignWizard',
 				[
-					'restPath'    => esc_url( $this->plugin->onboarding_rest_controller->get_base_path() ),
-					'nonce'       => wp_create_nonce( 'wp_rest' ),
-					'themeStatus' => esc_html( $this->plugin->theme_status() ),
+					'restPath'      => esc_url( $this->plugin->onboarding_rest_controller->get_base_path() ),
+					'nonce'         => wp_create_nonce( 'wp_rest' ),
+					'themeStatus'   => esc_html( $this->plugin->theme_status() ),
+					'fontsRestPath' => esc_url( '/material-design/v1/design-assets/retrieve-fonts' ),
```

Comment:
Should we use `get_base_path` here? We have it [defined here](https://github.com/xwp/material-design-wp-plugin/blob/456b0ae643b086b545a9391c843f8727a302f51a/php/class-design-assets-rest-controller.php#L68) anyways